### DISCUSSION TITLE
Git sync

### DISCUSSION THREAD

**Discussion by FlorinBuffet** [requarks/wiki#6734] [Help / Questions] [answered]
Hi there
I tried to setup git sync but with SSH Private Key Mode "contents". Now i get this error:
`Fetching origin @@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@ @ WARNING: UNPROTECTED PRIVATE KEY FILE! @ @@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@ Permissions 0755 for '/data/secure/git-ssh.pem' are too open. It is required that your private key files are NOT accessible by others. This private key will be ignored. Load key "/data/secure/git-ssh.pem": bad permissions git@github.com: Permission denied (publickey). fatal: Could not read from remote repository. Please make sure you have the correct access rights and the repository exists. error: could not fetch origin`
How can i fix this?
Best Regards
Florin

**Comment by fnafox** [comment]
You need to secure your private key like the message says.
Assuming your key is located at the listed location, log in to the system and run the following:
`chmod 600 /data/secure/git-ssh.pem`
You will also need to change the ownership to the user your wiki process is running as. Assuming that user is wiki, then:
`chown wiki:wiki /data/secure/git-ssh.pem`

If your wiki is running as a docker container, then you need to change these permissions at the location where the file was mounted from.

**Comment by FlorinBuffet** [comment, marked as answer]
did it over pat now
